examples/src/main/java/com/mapbox/navigation/examples/core/DebugMapboxNavigationKt.kt

```diff
@@ -153,6 +153,7 @@ class DebugMapboxNavigationKt : AppCompatActivity(), OnMapReadyCallback,
         val newOptions =
                 options.toBuilder()
                         .onboardRouterConfig(onboardRouterConfig)
+                        .navigatorPollingDelay(1000L)
```

Comment:
nit: use default instead of set explicit here. unless it's testing not the default

Reply:
Yeah it was changed for testing purposes 👍 cc @mskurydin @Lebedinsky